### PR TITLE
Messaging backend

### PR DESCRIPTION
**Pull request by otakaran**
Messaging now uploading into the database with proper parameters using email as a unique id right now.

1. Send a message inside the application
2. Take note of the message-id in the dubug console
3. Go to: https://console.firebase.google.com/project/mustang-messenger-41f23/firestore/ and see that the message is now present
4. profit

Will create a new issue on the todo of verifying the userTo parameter.

Time spent: 1.5 hours

### CODE REVIEW

**Comment by ethanchoi1** [comment]
I tested your implementation. I was successfully able to send a message and see it appear on the database. Great work!

I think the user verification is a good future issue, good thinking. I've made an issue for this and added it to our backlog.

File: src/fire/sendMessageFire.js [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
 function sendMessageFirebase(event) {
-  // nothing yet
+  // Proccess incoming data from the form
+  event.preventDefault();
+  const data = new FormData(event.target);
+  var messageTo = (data.get('messageTo'));
+  var messageBody = (data.get('messageBody'));
+  console.log("Sending a message into db...")
+  console.log("messageTo: ", messageTo);
+  console.log("messageBody: ", messageBody);
```
Comment (ethanchoi1) [inline src/fire/sendMessageFire.js:12]: Once we are able to display messages, remember to remove these. Fine for now.

**Review by ethanchoi1 — APPROVED** [review]
Code looks great! I didn't really have too many issues with it, looks good to me. Excellent work!

File: src/fire/sendMessageFire.js (hunk 1 above)
Comment (otakaran) [reply src/fire/sendMessageFire.js:12]: Noted.

**Comment by otakaran** [comment]
Awesome, giving some time for @corimstyle to look it over if he's available. Otherwise will merge this into main tonight.

File: src/fire/sendMessageFire.js (hunk 1 above)
Comment (otakaran) [reply src/fire/sendMessageFire.js:12]: Will create an issue to clean up debug prints before the "final" release.

**Pull request merged**